Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Build
+FROM rust:1.73-alpine3.18 AS build
```

Comment:
I recommend using image hashes instead of versions. Otherwise a future push to the image may break the build. Renovate should take care of updates and we should have a build pipeline to validate that building and running works

Reply:
Can you suggest the change?

Reply:
it seems a bit extreme give we use a widely used image, so I will hold off on this for now.